types.go

```diff
@@ -56,3 +56,17 @@ type PlotExpr struct {
 func (expr PlotExpr) String() string {
```

Comment:
Do we actually still use this, or should we just replace it with the code below and not have a separate "TruncatedString" function?

Reply:
it is used https://github.com/bugsnag/promalert/blob/7d9beb1d6d2172d2dc998aa0effa152c75748132/plotter.go#L32 which is why i didnt replace it